geometry_msgs/Point position
geometry_msgs/Point velocity
float64 yaw